### PR TITLE
feat: support lrucache

### LINKED ISSUE #8: feature: need to implement `lrucache`

### PR DESCRIPTION
**Pull request by nic-chen**
close #8

### CODE REVIEW

File: lua/deps/resty/lrucache/pureffi.lua [hunk 1]
```diff
@@ -0,0 +1,606 @@
+-- Copyright (C) Yichun Zhang (agentzh)
```
Comment (membphis) [inline lua/deps/resty/lrucache/pureffi.lua:1]: remove this file

File: lua/deps/resty/lrucache.lua [hunk 2]
```diff
@@ -0,0 +1,334 @@
+-- Copyright (C) Yichun Zhang (agentzh)
```
Comment (membphis) [inline lua/deps/resty/lrucache.lua:1]: added to `licenses` folder

**Review by membphis — CHANGES_REQUESTED** [review]

**Comment by tokers** [comment]
I think we should build the tests from now on.

**Comment by membphis** [comment]
> I think we should build the tests from now on.

yes, related issue: https://github.com/api7/envoy-apisix/issues/16

**Review by membphis — APPROVED** [review]

**Pull request merged**